Assets/MixedRealityToolkit.SDK/Inspectors/UX/Interactable/PressableButtonInspector.cs

```diff
@@ -183,60 +234,71 @@ private void OnSceneGUI()
             Handles.DrawDottedLine(targetStartPlane[1], targetEndPlane[1], 2.5f);
             Handles.DrawDottedLine(targetStartPlane[2], targetEndPlane[2], 2.5f);
             Handles.DrawDottedLine(targetStartPlane[3], targetEndPlane[3], 2.5f);
-
-            serializedObject.ApplyModifiedProperties();
         }
 
-        private void DrawPlaneAndHandle(Vector3[] plane, Vector3 planeSize, float handleSize, ref Vector3 position, string label)
+        private void DrawPlaneAndHandle(Vector3[] plane, Vector3 planeSize, ref float zPosition, Vector3 cagePosition, string label, bool editingEnabled)
```

Comment:
just curious, why are you modifying zPosition as a ref variable instead of just returning the new zPosition as a return value for the method?

Reply:
Cruft, will remove.